activestorage/lib/active_storage/service/registry.rb

```diff
@@ -14,12 +14,13 @@ def fetch(name)
         else
           if block_given?
             yield key
+            nil
```

Comment:
Previously, calling `Registry#fetch` with a block would return the block's return value. It should return a `Service`, or `nil` if no such service exists. (This is an internal API marked with `:nodoc:` so I think it's fine to change.)